internal/statemachine/tests_helper_test.go

```diff
@@ -130,6 +131,10 @@ func NewMockExecCommand() *mockExecCmd {
 }
 
 func (m *mockExecCmd) Command(cmd string, args ...string) *exec.Cmd {
+	// Replace all newlines in each arg with a space
+	for i, arg := range args {
+		args[i] = strings.ReplaceAll(arg, "\n", " ")
+	}
```

Comment:
Is this still needed?

Reply:
No :)

Reply:
It may enabling a nicer output but that might be confusing when trying to debug a command. If we were to pass a multi-line string on purpose, that might be confusing to see a different output in the test output.